### PR TITLE
Various post display improvements

### PR DESCRIPTION
**Pull request by MrPetovan**
This PR includes two distinct post display improvements:
1. Post direction icon change to reflect comment better, with an additional base English language change
2. For frio only, the reshare information is now shown at the top of posts in the network page. This increases consistency with other popular social media platforms such as Mastodon and Twitter that both show the resharer on top of the reshared posts.

### CODE REVIEW

**Comment by tobiasd** [comment]
Could you please regenerate the `messages.po` file so that the translation pipe can pick up the changes?

**Comment by MrPetovan** [comment]
Ha, I did but forgot to commit the changes, thanks for the reminder.

File: view/theme/frio/templates/sub/direction.tpl [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	{{if $direction.direction == 1}}
 		<i class="fa fa-inbox" aria-hidden="true" title="{{$direction.title}}"></i>
 	{{elseif $direction.direction == 2}}
-		<i class="fa fa-download" aria-hidden="true" title="{{$direction.title}}"></i>
+		<i class="fa fa-comment-o" aria-hidden="true" title="{{$direction.title}}"></i>
```
Comment (annando) [inline view/theme/frio/templates/sub/direction.tpl:7]: `2` is also used for indicating that the post had been fetched. So I suggest creating a direction "5" with the "comment" icon.

Comment (MrPetovan) [reply view/theme/frio/templates/sub/direction.tpl:7]: Ok.

**Comment by MrPetovan** [comment]
I added `direction = 5` but this means `direction = 2` isn't used anymore. Not sure how to qualify a post that was fetched.

**Pull request merged**